apps/shared/scripts/generate-gallery-tokens.mjs

```
@@ -0,0 +1,381 @@
+/**
+ * Entry point: generate and write, or verify with `--check`.
+ * Exits non-zero in check mode when the committed file is stale.
+ */
+function main() {
+	// The resolution reads the shell package's SOURCE (styles.ts and the
+	// surface component .tsx files). Standalone app repos vendor only the
+	// COMPILED shell (.rocketride/shell), so a faithful regeneration — and
+	// therefore drift-checking — is only possible in the monorepo. Keep the
+	// committed map as-is and skip here.
+	if (!existsSync(STYLES_FILE)) {
+		console.log('shell source not present (standalone repo) - keeping committed tokenUsage.generated.ts');
```

ESLint

[error] 'console' is not defined.

(no-undef)